bundles/AdminBundle/Resources/public/js/pimcore/document/editables/areablock.js

```diff
@@ -1030,10 +1030,11 @@ pimcore.document.editables.areablock = Class.create(pimcore.document.area_abstra
             var toolbar = new Ext.Window({
                 title: areaBlockToolbarSettings.title,
                 width: areaBlockToolbarSettings.width,
+                height: Ext.getBody().getViewSize().height - 15,
                 border:false,
                 shadow: false,
                 resizable: false,
-                autoHeight: true,
+                scrollable: true,
```

Comment:
```suggestion
                scrollable: 'y',
```
Scrolling just vertically should be sufficient I think 😉